io.catenax.vehicle.product_description/RELEASE_NOTES.md

```diff
@@ -3,6 +3,13 @@ All notable changes to this model will be documented in this file.
 
 ## [Unreleased]
 
+## [3.0.0] - 2023-05-16
```

Comment:
still not sure, if this should be the date of the MS3 release. Should be clarified by @bs-jokri